pkg/kubelet/certificate/kubelet.go

```diff
@@ -300,7 +303,9 @@ func (m *kubeletServerCertificateDynamicFileManager) Current() *tls.Certificate
 // Start starts watching the certificate and key files
 func (m *kubeletServerCertificateDynamicFileManager) Start() {
 	var ctx context.Context
-	ctx, m.cancelFn = context.WithCancel(context.Background())
+	// Use context.TODO() because we currently do not have a proper context to pass in.
+	// This should be replaced with an appropriate context when refactoring this function to accept a context parameter.
+	ctx, m.cancelFn = context.WithCancel(context.TODO())
```

Comment:
can we initiate the TODO from the callsite? I am pretty sure Start will need to accept ctx going forward